File: output/ChatGPT/codenet/Java/C++/s577060895.cpp
#include <iostream>
#include <cmath>

int main() {
    long n;
    std::cin >> n;

    double sqrt = std::sqrt(n);
    double nearNum = std::floor(sqrt);

    long ans = std::pow(nearNum, 2);

    std::cout << ans << std::endl;

    return 0;
}

File: output/CodeGen/codenet/Java/Python/s577060895.py
def main():
    sc = Scanner(System.in)
    n = int(sc.nextLine())
    sqrt = math.sqrt(n)
    nearNum = int(math.floor(sqrt))
    ans = int(math.pow(nearNum, 2))
    print(ans)
    sc.close()

if __name__ == '__main__':
    main()